jangjuseong/.env.sample

```
@@ -0,0 +1,11 @@
+DATABASE_URL = URL
```

우선 sample 파일은 다른 협업 개발자도 이해하기 쉽게 작성을 하되 중요한 값들은 노출되지 않도록 작성해주면 됩니다!

아래와 같이 할당할 수 있는 변수명을 작성해줌으로써 어떤 값들이 포함되는지 공유할 수 있습니다.
```suggestion
DATABASE_URL = "rdbms://username:password@host:port/database-name"
```